go.mod

```diff
@@ -33,10 +33,14 @@ require (
 	github.com/ory/go-acc v0.2.6
 	github.com/pavius/impi v0.0.3
 	github.com/securego/gosec/v2 v2.5.0
+	github.com/signalfx/defaults v1.2.2-0.20180531161417-70562fe60657
+	github.com/signalfx/signalfx-agent v0.0.0-00010101000000-000000000000
```

Comment:
Is `v0.0.0-00010101000000-000000000000` a correct version number of SA that we want to use? The latest version appears to be `v5.7.1`, but we are not referencing it here.

Reply:
It's the pseudo version that doesn't refer to an actual commit but is inserted by go when using local paths: https://golang.org/doc/go1.12#modules.  The subsequent replace directive will always apply for our builds so I don't believe it really matters what version is specified here.